### PR TITLE
[full-ci][tests-only]Use cached web in drone pipelines

### PR DESCRIPTION
**Pull request by amrita-shrestha**
### Description
This PR has implemented using the cached web in the web pipeline and remove unnecessary step from pipelines

### Related issue
[cache build-web-integration-app step in web](https://github.com/owncloud/QA/issues/762)

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingFolderAdvPermsGrp</strong> https://drone.owncloud.com/owncloud/web/28018/40/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L54.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L55.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L56.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L57.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L58.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L59.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L60.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L61.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L62.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L63.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L64.png
  

### webUISharingFolderAdvancedPermissionsGroups-shareAdvancePermissionsGroup_feature-L65.png
  


</p></details>

File: .drone.star [hunk 1]
```diff
@@ -1411,7 +1409,10 @@ def acceptance(ctx):
                             else:
                                 ## Configure oc10 and web with oauth2 and web Service
                                 steps += setUpOauth2(params["oc10IntegrationAppIncluded"], True)
-                                services += webService()
+
+                                ## run build-web-integration for oc10IntegrationAppIncluded only
```
Comment (saw-jan) [inline .drone.star:1413]: comment seems unrelated
```suggestion
                                ## web service is not required for web-oc10-integration
```

File: .drone.star [hunk 2]
```diff
@@ -1210,6 +1209,7 @@ def e2eTests(ctx):
         steps = skipIfUnchanged(ctx, "e2e-tests") + \
                 restoreBuildArtifactCache(ctx, "yarn", ".yarn") + \
                 restoreBuildArtifactCache(ctx, "playwright", ".playwright") + \
+                restoreBuildArtifactCache(ctx, "web-dist", "dist") + \
                 installYarn() + \
```
Comment (grgprarup) [inline .drone.star:1213]:
```suggestion
                installYarn() + \
                restoreBuildArtifactCache(ctx, "web-dist", "dist") + \
```
IMO this will be the flow for running steps.

**Comment by grgprarup** [comment]
@amrita-shrestha  Also, don't forget to check and refactor in other owncloud repos. You can add a checklist to the tagged issue.

File: .drone.star (hunk 2 above)
Comment (saw-jan) [reply .drone.star:1213]: I won't matter though

**Comment by saw-jan** [comment]
this line can also be remove. I think it has no use
https://github.com/owncloud/web/blob/46310b48f3bc70002c45603f30a5f1a0bc4457aa/.drone.star#L1370

**Review by grgprarup — COMMENTED** [review]
LGTM :+1:

**Review by saw-jan — COMMENTED** [review]
LGTM :+1:

**Review by phil-davis — APPROVED** [review]
